phpstan.neon

```diff
@@ -69,6 +69,11 @@ parameters:
         - identifier: missingType.callable
           path: tests/Fixture/Maker/expected/
 
+        # we're currently running static analysis with PHP 8.3
```

Comment:
I tried to use php 8.4 for the static analysis workflow, but psalm is complaining for stuff I don't want to fix now :sweat_smile: